QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java

```diff
@@ -753,7 +755,40 @@ public void saveDraft() {
 
```

Comment:
This line can be moved to 785, and replace the Toast

So this way of storing the draft will only be used if there are no recipients